cmd/conformance/main.go

```diff
@@ -221,12 +221,31 @@ func main() {
 		// Load trust root
 		tr := getTrustedRoot()
 
-		// Verify bundle
-		sev, err := verify.NewSignedEntityVerifier(tr, verify.WithTransparencyLog(1), verify.WithSignedCertificateTimestamps(1))
+		verifierConfig := []verify.VerifierOption{}
+		verifierConfig = append(verifierConfig, verify.WithSignedCertificateTimestamps(1))
+
+		// Check bundle and trusted root for signed timestamp information
+		bundleTimestamps, err := b.Timestamps()
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		if len(tr.TSACertificateAuthorities()) > 0 && len(bundleTimestamps) > 0 {
```

Comment:
Mentioned this on slack, but what if the trust root specifies a TSA certificate and the bundle contains a signed timestamp issued by a different TSA cert? This conformance check expects that there is at least one 1-1 mapping between trusted TSA certs and signed timestamps.

As is I think this is fine for now, because the "fix" is another policy flag to say that the verifier just needs a timestamp.

Reply:
Yes, I think the scope of this PR is "allow bundles relying on valid signed timestamps from a TSA to pass conformance testing", which previously was not the case.